src/kata5.reachDestination.js

```diff
@@ -1,3 +1,12 @@
-const reachDestination = (distance, speed) => {};
+// should return a string "I should be there in (time) hours."
+// 2 inputs (distance & speed) both of which are known
+// work out (time) by distance / speed.
+// (time) rounded to the nearest 0.5
+
+const reachDestination = (distance, speed) => {
+  const time = Math.round((distance / speed) * 2)/2;
+
+  return "I should be there in " + time + " hours.";
```

Comment:
I would use a template literal for this, i.e. `return I should be there in ${time} hours.";`